src/state/nftMarket/helpers.ts

```diff
@@ -223,6 +223,69 @@ export const getNftsMarketData = async (
   }
 }
 
+export const getAllPancakeBunniesLowestPrice = async (bunnyIds: string[]): Promise<Record<string, number>> => {
```

Comment:
this and `getAllPancakeBunniesRecentUpdatedAt` is used to get all lowest prices and latest updatedAt from subgraph for all existing bunnyIds
Needed for sorting.

This is somewhat duplicated when displaying the lowestPrice on the card itself (it makes another request). Decided not to touch that for now.